src/test/java/org/broadinstitute/consent/http/db/DataAccessRequestDAOTest.java

```diff
@@ -762,6 +779,40 @@ void testFindAllDataAccessRequestDatasArchived() {
     assertEquals(1, returnedDAR.size());
   }
 
+  @Test
+  void testFindDARsByDateRange() throws InterruptedException {
+    DarCollection darCollection = createDarCollection();
+    darCollection.getDars().keySet().forEach((referenceId) -> {
+      dataAccessRequestDAO.updateDraftToSubmittedForCollection(darCollection.getDarCollectionId(),
+          referenceId);
+    });
+    Timestamp oneYearAgo = Timestamp.from(
+        Instant.ofEpochMilli(new Date().getTime() - DataAccessRequest.EXPIRATION_DURATION_MILLIS));
+    Timestamp oneMinuteInTheFuture = Timestamp.from(
+        Instant.ofEpochMilli(new Date().getTime() + (60 * 1000)));
+    List<DataAccessRequest> dars = dataAccessRequestDAO.findSubmittedDarsByTimeRange(oneYearAgo,
+        oneMinuteInTheFuture);
```

Comment:
I think `Instant` would be easier to read here
```suggestion
    Instant oneYearAgo = Instant.now().minus(365, ChronoUnit.DAYS);
    Instant oneMinuteInTheFuture = Instant.now().plus(1, ChronoUnit.MINUTES);
    List<DataAccessRequest> dars = dataAccessRequestDAO.findSubmittedDarsByTimeRange(Timestamp.from(oneYearAgo),
        Timestamp.from(oneMinuteInTheFuture));
```